_rules/aria-state-or-property-permitted-5c01ea.md

```diff
@@ -64,6 +64,8 @@ The presence of prohibited ARIA attributes is often the result of a developer us
 
 In HTML, there are language features that do not have corresponding implicit WAI-ARIA semantics. As per [ARIA in HTML](https://www.w3.org/TR/html-aria/), those elements can have [global states or properties][global]. Some of those elements can also have [inherited][], [supported][], or [required][] [states][state] or [properties][property] that correspond to a [WAI-ARIA role](https://www.w3.org/TR/wai-aria-1.2/#introroles). For example, the `audio` element has no corresponding ARIA semantics but it can have [inherited][], [supported][], or [required][] [states][state] or [properties][property] of the [`application` role](https://www.w3.org/TR/wai-aria-1.2/#application).
 
+This rule only applies to [explicitly set][aria set explicit] attributes since these have been provided by authors and using the wrong one is therefore an author issue. If an [implicitly set][aria set implicit] attribute is incorrect, this is not necessarily an author error, e.g., it can be the result of re-purposing an existing element.
```

Comment:
Would this not still be a responsibility of the author to ensure the attributes are correct? It could still lead to violations, does the [ARIA state or property has valid value](https://www.w3.org/WAI/standards-guidelines/act/rules/6a7281/) check for this? If so, I think we should just call that out directly.

Reply:
If only it was so simple 😞 

`<input type="checkbox" role="button" aria-pressed="false" />` is [allowed by ARIA](https://www.w3.org/TR/html-aria/#el-input-checkbox) (as far as I can tell), but has an implicit `aria-checked` set by `checked` (automagically set on `<input type="checkbox" />`), which is not allowed on a role of `button`…